addons/resupply/config.cpp

```diff
@@ -0,0 +1,183 @@
+#include "script_component.hpp"
+
+class CfgPatches
+{
+    class ADDON
+    {
+        name = QUOTE(COMPONENT);
+        units[] = {"Box_SWS_TeamBox","SWS_Ammo_SupplyPod_Empty","SWS_Ammo_SupplyPod_Basic","SWS_Ammo_SupplyPod_Medical","SWS_Ammo_SupplyPod_Howdy"};
+        weapons[] = {};
+        requiredVersion = REQUIRED_VERSION;
+        requiredAddons[] = {
+            "OPTRE_Misc",
+            "SWS_Weapons",
```

Comment:
```suggestion
            "sws_weapons",
```
Won't break anything, but the addon names are lowercased.